snapcraft/internal/sources.py

```diff
@@ -80,6 +82,28 @@ def __init__(self, source, source_dir, source_tag=None,
         self.source_branch = source_branch
 
 
+class FileBase(Base):
+
+    def pull(self):
+        if common.isurl(self.source):
+            self.download()
+        else:
+            shutil.copy2(self.source, self.source_dir)
+
+        self.provision(self.source_dir)
```

Comment:
This doesn't seem to be implemented in this class. Should it be, and raise a NotImplemented or something?